entities/params/sendcampaign.go

```diff
@@ -0,0 +1,15 @@
+package params
+
+import "strings"
+
+// SendCampaign represents request body for POST /api/campaigns/id/start
+type SendCampaign struct {
+	Ids                 []int64           `form:"segment_id[]" validate:"gt=0,dive,required"`
+	Source              string            `form:"source" validate:"required,email,max=191"`
+	FromName            string            `form:"from_name" validate:"required,max=191"`
+	DefaultTemplateData map[string]string `form:"default_template_data" validate:"dive,required"`
```

Comment:
@Dzalevski we should validate the DefaultTemplateData as we do in [entities/subscribers.go](https://github.com/mailbadger/app/blob/009f4c00d3ed0559f600527d514d8806474f8a2c/entities/subscriber.go#L78)

We can leave the `dive,required` since we require all keys to have values. But we'll need to validate the keys as well.